pyGHDL/libghdl/vhdl/nodes.py

```diff
@@ -1096,6 +1096,7 @@ class Iir_Kinds:
     ]
 
 
+
```

Comment:
Execute `./scripts/update_py_bindings.sh` right before committing your changes, in order to have Python sources regenerated. Then, run `black pyGHDL` to have consistent formatting. Black is a Python package, which you can install through pip.

Reply:
OK.  I believe I re-ran those and updated.  Let's see.

Reply:
It seems that you setup keeps removing some lines in `pyGHDL/libghdl/vhdl/nodes.py`. Can you check that black is up to date (CI uses `black-21.9b0`) and that no other editor/IDE settings are modifying the sources?

Reply:
OK, I'll keep this unresolved until we figure it out.

I think this latest push might have fixed?  Let's wait for the CI to tell us.

Reply:
Yes, it did! Thanks!